data/decoder.js

```diff
@@ -0,0 +1,6 @@
+var Base64 = require('js-base64').Base64;
+var data = require('./secret.csv')

```

Comment:
what is this file for? if it's not being used / was for prototyping may be better to remove